### PR TITLE
Add FAQ article eu_dcc_remove

### PR DESCRIPTION
**Pull request by larswmh**
chrome_qECAYsX5bE
chrome_8qtiS2eRyG

This FAQ article was originally planned to only cover a guide on how to remove vaccination certificates, but was now generalized to cover any certificate removal

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -172,6 +172,15 @@
...
                         ]
+                    },
+                    {
+                        "title": "How do I remove a certificate from the Corona-Warn-App?",
+                        "anchor": "eu_dcc_remove",
+                        "active": true,
+                        "textblock": [
+                            "Using this step-by-step guide, you can remove certificates from the Corona-Warn-App. Please note that the first three steps apply to both iOS and Android, but only one of the last two steps needs to be followed, depending on which operating system matches yours. Vaccination certificates, test certificates and recovery certificates are mentioned as 'Certificates', since the process of removing them is identical.",
+                            "<ul><li>Tap on the 'Certificates' tab</li><li>Tap on the certificate you would like to remove</li><li>Scroll down to and tap on the tile that matches the certificate you want to delete.</li><li>iOS: Tap on 'Delete Certificate' at the bottom of the screen</li><li>Tap on the three-dot symbol on the top right of the screen, and then tap on 'Remove'</li></ul>"
```
Comment (MikeMcC399) [inline src/data/faq.json:182]:
```suggestion
                            "<ul><li>Tap on the 'Certificates' tab</li><li>Tap on the certificate you would like to remove</li><li>Scroll down to and tap on the tile that matches the certificate you want to delete.</li><li>iOS: Tap on 'Delete Certificate' at the bottom of the screen</li><li>Android: Tap on the three-dot symbol on the top right of the screen, and then tap on 'Remove'</li></ul>"
```

Comment (Ein-Tim) [inline src/data/faq.json:182]: Missing reference to Android

```suggestion
                            "<ul><li>Tap on the 'Certificates' tab</li><li>Tap on the certificate you would like to remove</li><li>Scroll down to and tap on the tile that matches the certificate you want to delete.</li><li>iOS: Tap on 'Delete Certificate' at the bottom of the screen</li><li>Android: Tap on the three-dot symbol on the top right of the screen, and then tap on 'Remove'</li></ul>"
```

**Review by Ein-Tim — CHANGES_REQUESTED** [review]
One thing (:

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Missing Android qualifier in English text.

File: src/data/faq.json (hunk 1 above)
Comment (Ein-Tim) [reply src/data/faq.json:182]: Already covered by @MikeMcC399 suggested changes in https://github.com/corona-warn-app/cwa-website/pull/1607#discussion_r685881615

**Review by Ein-Tim — APPROVED** [review]

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
@MikeMcC399 @Ein-Tim @larswmh Thanks everybody!

**Pull request merged**